ompi/mca/pml/ob1/pml_ob1_recvfrag.c

```diff
@@ -484,9 +485,11 @@ void mca_pml_ob1_recv_frag_callback_match (mca_btl_base_module_t *btl,
         return;
     }
     comm = (mca_pml_ob1_comm_t *)comm_ptr->c_pml_comm;
+    assert(NULL != comm);
 
     /* source sequence number */
     proc = mca_pml_ob1_peer_lookup (comm_ptr, hdr->hdr_src);
+    assert(NULL != proc);
```

Comment:
For symmetry I think we should also add the assertion on L1080 & L1084

Reply:
these are unnecessary, a valid communicator cannot have the c_pml_comm set to zero, nor an empty proc